static/js/tour.js

```diff
@@ -118,29 +103,48 @@ function startTour() {
                 content: "Select the reference genome corresponding to the GWAS data you have uploaded. For the example GWAS file, select the 'GRCh38/hg38' option."
             },
             {
-                element: "#pvalInput",
+                element: "#pvalContainer",
                 title: "Enter p-value cutoff",
                 content: "Enter the p-value cutoff for odds ratios you wish to include in your polygenic risk score \
-                using the two text boxes."
+                using the two text boxes. The default value is 1.0x10^-5."
+            },
+            {
+                element: "#superPopContainer",
+                title: "Select super population of individual(s) in your VCF file",
+                content: "This will be used to perform LD clumping on your data (see the About \
+                page). If you are unsure about the super population of the individual(s) in your file, \
+                choose the one you believe to be the most accurate."
+            },
+            {
+                element: "#ldContainer",
+                title: "Select LD Clumping Type",
+                content: "Choose whether LD clumping should be performed sample-wide or by individual. For \
```

Comment:
Maybe we should direct them to an explanation of the differences between the two.